cockatrice/src/setsmodel.h

```diff
@@ -43,7 +43,8 @@ class SetsModel : public QAbstractTableModel {
     QStringList mimeTypes() const;
     void swapRows(int oldRow, int newRow);
     void toggleRow(int row, bool enable);
-    void toggleAll(bool enable);
+    void toggleRow(int row);
+    void toggleAll(int rowToCopyStatusFrom);
```

Comment:
The current behavior:
`if (row1 isEnabled) { disableAllRows() } else { enableAllRows() }`